appveyor.yml

```diff
@@ -17,12 +16,13 @@ install:
     }
 
     git submodule update --recursive --force
-
     nuget restore GitHubVS.sln
 build_script:
-- cmd: msbuild "GitHubVS.sln" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /p:Configuration=Release /p:DeployExtension=false /verbosity:minimal /p:VisualStudioVersion=14.0
+- ps: scripts\build.ps1 -AppVeyor -BuildNumber:$env:APPVEYOR_BUILD_NUMBER
 test_script:
-- ps: >-
-    scripts\Run-Nunit.ps1 TrackingCollectionTests 180 Release -AppVeyor
-
-    scripts\Run-Xunit.ps1 UnitTests 180 Release -AppVeyor
\ No newline at end of file
+- ps: scripts\test.ps1 -AppVeyor
```

Comment:
@shana do we want to use appveyor's automatic test detection? https://www.appveyor.com/docs/running-tests/#selecting-assemblies-andor-categories-to-test

Reply:
Yeah, we just need to consolidate all the tests in one place so we can have AppVeyor run them with the correct categories ignored. We also need to consolidate the test categories to be the same for everyone.